CHANGELOG.md

```diff
@@ -1,5 +1,34 @@
 # CHANGELOG
 
+## `v46.1.0`
+
+### New Packages
+
+| Package Name | API Version |
+| ---: | :---: |
+| hybridcompute | 2019-12-12<br/>2020-08-02 |
+| mariadb | 2020-01-01 |
+| netapp | 2020-06-01 |
+| operationalinsights | 2020-08-01 |
+| subscription | 2020-09-01 |
+
+### Updated Packages
+
+| Package Name | API Version |
+| ---: | :---: |
+| datafactory | 2018-06-01 |
+| mysql | 2020-01-01 |
+
+## Removed Packages
+
+| Package Name | API Version |
+| ---: | :---: |
+| hybridcompute | 2019-12-12 |
```

Comment:
Can this wait for v47?  While I think it's ok to make breaking changes within preview packages, I don't think we should replace them with their stable counterpart outside of a major version bump.

Reply:
Yeah, reasonable.

Actually I am cleaning those cases that a stable version is placed under the preview directory by mistake.

I will move this package to the next major version with all the others